pkg/costattribution/manager.go

```diff
@@ -21,6 +21,7 @@ const (
 	defaultTrackerName = "cost-attribution"
 	missingValue       = "__missing__"
 	overflowValue      = "__overflow__"
+	usagePrefix        = "attributed_"
```

Comment:
I'm not sure why is this called "usagePrefix", but I know naming is hard.
Maybe it could be `exportedAttributedLabelPrefix`?